heroku/cli
# refactor(redis-v5): Move 'redis:maxmemory' to CLI

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001axXVwYAM/view

This converts `redis:maxmemory` to oclif and updates its associated tests. It also includes a conversion of the `getRedisAddon()` lib function.

### Testing

1) Run `./bin/run redis:maxmemory -a APP -p POLICY` with this branch pulled down with an app that has redis on it to update the policy to a valid POLICY (refer to `help` for valid policies)
2) Verify the command updates the policy by running `heroku redis:info -a APP`
3) Compare the results with prod cli command and make sure things look 👍
